### DISCUSSION TITLE
是否考虑支持默认数据源放开Debug模式

### DISCUSSION THREAD

**Discussion by yinlianghui** [steedos/steedos-platform#3213] [Ideas]
- 现在上线了共享和限制规则经常要调式过滤条件，如果能开启Debug模式会比较方便调式问题
- 原来我在外部数据源列表放开过默认数据源放开Debug模式是有效果的。
- 现在已经无法配置默认数据源放开Debug模式了
<img width="1023" alt="wecom-temp-499182b3fa721ea83dd79fc0367975b6" src="https://user-images.githubusercontent.com/6219465/149748027-ee589e6c-6b7e-442d-8abd-5ce5a8f9916c.png">

**Comment by baozhoutao** [comment]
默认数据源的debug是啥效果，是否给mongodb做过debug扩展？
关系型数据源的debug功能来自typeorm logging。
